ValkeyLDAP.md

```diff
@@ -169,6 +145,24 @@ The list of configuration options is the following:
   - `ldap.search_scope`: the LDAP search scope. Possible values `base`, `one`, `sub` (default: `sub`)
   - `ldap.search_dn_attribute`: the attribute that contains the DN of the user entry (default: `"entryDN"`)
 
+### INFO Command
+
+This module extends the `INFO` command output with a new section to show the status information of each LDAP server configured in `ldap.servers` configuration option.
+
+The new section is named `ldap_status` and contains a dictionary field per each server.
+
+Each dictionary field may contain the following fields:
+* `url`: the URL of the server as specified in `ldap.servers` configuration options.
+* `status`: the possible values for this field are `healthy`, or `unhealthy`; the `healthy` value denotes that the server is connectable and responds to LDAP operations, and the `unhealthy` value denotes that the server cannot process LDAP operations.
+* `ping_time`: shows the RTT for an simple LDAP operation in milliseconds; this field is only visible for `healthy` servers.
+* `error`: a string value with the cause for setting the server as `unhealthy`ç this field is only visible for `unhealthy` servers.
+
+Example of the output returned by `INFO`:
+```
+# ldap_status
+ldap_server_0:url=ldap://<hostname>,status=unhealthy,error=<some error message>
```

Comment:
Is the status and error part of the URL?

In general, comma is allowed inside an URL (IIRC).

Reply:
I don't think the URL needs to be "exactly" right here. If you want the actual values, it's available in the config. I would consider just putting the hostname here.

Reply:
@madolson Like this?

```suggestion
ldap_server_0:host=<hostname>,status=unhealthy,error=<some error message>
```